docs/update.md

```diff
@@ -95,40 +95,53 @@ You can upgrade the Operator and CRD as follows, considering the Operator uses
+    
+2. Update the [Custom Resource Definition :octicons-link-external-16:](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+    for the Operator, taking it from the official repository on Github.
 
     ``` {.bash data-prompt="$" }
-    $ helm upgrade my-operator percona/pg-operator --version {{ release }}
+    $ kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/crd.yaml
     ```
 
-    The `my-operator` parameter in the above example is the name of a [release object :octicons-link-external-16:](https://helm.sh/docs/intro/using_helm/#three-big-concepts)
-    which which you have chosen for the Operator when installing its Helm chart.
+3. Upgrade the Operator deployment
 
-    If the Operator was installed with some [customized parameters :octicons-link-external-16:](https://github.com/percona/percona-helm-charts/tree/main/charts/pg-operator#installing-the-chart), you should list these options in the upgrade command.   
-    
-    !!! note
-    
-        You can get list of used options in YAML format with the `helm get values my-operator -a > my-values.yaml` command, and this file can be directly passed to the upgrade command as follows:
+    === "With default parameters"
+
+        To upgrade the OPerator installed with default parameters, use the following command: 
 
         ``` {.bash data-prompt="$" }
-        $ helm upgrade my-operator percona/pg-operator --version {{ release }} -f my-values.yaml
+        $ helm upgrade my-operator percona/pg-operator --version {{ release }}
         ```
 
+        The `my-operator` parameter in the above example is the name of a [release object :octicons-link-external-16:](https://helm.sh/docs/intro/using_helm/#three-big-concepts)
+        which which you have chosen for the Operator when installing its Helm chart.
+
+    === "With customized parameters"
+
+        If you installed the Operator with some [customized parameters :octicons-link-external-16:](https://github.com/percona/percona-helm-charts/tree/main/charts/pg-operator#installing-the-chart), list these options in the upgrade command.   
+    
+        1. Get the list of used options in YAML format :
+        
+            ```{.bash data-prompt="$" }
+            $ helm get values my-operator -a > my-values.yaml
+            ``` 
+        
+        2. Pass these options to the upgrade command as follows:
+
+            ``` {.bash data-prompt="$" }
+            $ helm upgrade my-operator percona/pg-operator --version {{ release }} -f my-values.yaml
+            ```
+    During the upgrade, you may see a warning to manually apply the CRD if it has the outdated version. In this case, repeat step 2 to upgrade the CRD and then step 3 to upgrade the deployment.
```

Comment:
I think that people won't get a warning message after doing the step 2, the warning is for the case they didn't follow this documentation (or they not applied the correct CRD version)